### ISSUE TITLE
Operators are sometimes split into two keyword elements

### ISSUE THREAD

**Issue by engelfrost** [atom/language-javascript#259]
The expression `a !== b` is split into 
```
<span class="source js">
    a
    <span class="keyword operator js">!=</span>
    <span class="keyword operator js">=</span>
    b
</span>
```
instead of:
```
<span class="source js">
    a
    <span class="keyword operator js">!==</span>
    b
</span>
```
This prevents ligatures from working in fonts like FiraCode.

**Comment by gusaiani** [comment]
+1

**Comment by MaximSokolov** [comment]
Was fixed in #225

**Comment by 50Wliu** [comment]
The fix *should* be in the latest 1.3.0 betas already.
